src/core/core.scale.js

```diff
@@ -326,9 +337,12 @@ var Scale = Element.extend({
 		me.beforeFit();
 		me.fit();
 		me.afterFit();
-		//
+		// Auto-skip
+		me._ticks = tickOpts.display && tickOpts.autoSkip ? me._autoSkip(me.getTicks()) : me.getTicks();
```

Comment:
can't we use `ticks` instead of `me.getTicks()` here?

Reply:
done